others/freivalds_algorithm.cpp

```diff
@@ -0,0 +1,169 @@
+/**
+ * @file
+ * @brief Implementation of [Freivalds' algorithm]
+ * (https://en.wikipedia.org/wiki/Freivalds%27_algorithm)
+ * @author [Darshana Sarma](https://github.com/Darshana-Sarma)
+ * @details
+ * Given three n × n matrices A, B, and
+ * C, a general problem is to verify whether AB=C
+ *
+ * The time Complexity of the algorithm is O(n^{2}).
+ */
+#include <array>     //   for std::array
+#include <cassert>   //   for assert
+#include <iostream>  //   for io operations
+
+/**
+ * @namespace others
+ * @brief Other algorithms
+ */
+namespace others {
+/**
+ * @namespace freivalds
+ * @brief Functions for [Freivalds']
+ * (https://en.wikipedia.org/wiki/Freivalds%27_algorithm) algorithm
+ */
+namespace freivalds {
+/**
+ * The function will check AB=C by applying Freidvald's algorithm
+ * @param A square matrix A

```

Comment:
Please provide a description for the template parameter.
```suggestion
 * @tparam N description
 * @param A square matrix A
```